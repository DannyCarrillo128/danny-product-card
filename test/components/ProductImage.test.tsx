import React from 'react';
import renderer from 'react-test-renderer';

import { ProductImage } from '../../src/components';
// import { product2 } from '../data/products';

describe('Product Image', () => {

  test('should show the component with the corresponding image', () => {
    const wrapper = renderer.create(
      <ProductImage className="custom-class" image="../test-image.jpg" />
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  /* test('should show the component with the product image', () => {
    const wrapper = renderer.create(
      <ProductCard product={ product2 }>
        {
          () => (
            <ProductImage />
          )
        }
      </ProductCard>
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  }); */

});
