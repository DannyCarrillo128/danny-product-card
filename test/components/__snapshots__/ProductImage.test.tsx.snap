// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Image should show the component with the corresponding image 1`] = `
<img
  alt="Product"
  className="productImg custom-class"
  src="../test-image.jpg"
/>
`;
